src/pages/home/Home.jsx

```diff
@@ -8,64 +8,117 @@ import FitnessCenterTwoToneIcon from '@mui/icons-material/FitnessCenterTwoTone';
 import EggAltTwoToneIcon from '@mui/icons-material/EggAltTwoTone';
 import MoodTwoToneIcon from '@mui/icons-material/MoodTwoTone';
 import { Container } from '@mui/material';
+import CardMedia from '@mui/material/CardMedia';
 
 const Home = () => {
   return (
-    <Container sx={{ mb: 10 }}>
-      <Box sx={{ backgroundImage: 'url(/assets/women.jpg)', p: 4, m: 0 }}>
-        <Stack
-          direction="column"
-          justifyContent="space-between"
-          spacing={8}
-          my={4}
-        >
-          <Typography variant="h4" color="white">
-            <div>Mysli na sebe i na mateřské dovolené.</div>
-            <div> Buď v kondici. </div>
-            <div> Buď svá. </div>
-          </Typography>
-        </Stack>
-        <Box
+    <>
+      <Box
+        sx={{
+          overflow: 'hidden',
+          height: 600,
+          position: 'relative',
+          '&:after': {
+            content: '""',
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            backgroundColor: 'rgba(0,0,0,0.5)',
+            zIndex: 1,
+          },
+        }}
+      >
+        <CardMedia
           sx={{
-            width: 350,
-            backgroundColor: 'primary',
-            color: 'white',
-            mt: 5,
-
-            p: 1,
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            objectFit: 'cover',
+            objectPosition: 'center 25%',
+            zIndex: 0,
+            height: '100%',
           }}
-        >
-          <p className="motivace">
-            Cvič tam, kde ti to vyhovuje a co tvé tělo potřebuje. Stravuj se
-            tak, aby Ti tvé tělo bylo vděčné a dosáhla jsi svých cílů. Vyplň
-            krátký dotazník a zjisti, jak na to.
-          </p>
-        </Box>
+          image="/assets/home.jpg"
+          title="home"
+          component="img"
+        />
+        <Container sx={{ mb: 10 }}>
+          <Stack
```

Comment:
Flex from `Stack` doesn't apply to `div` elements. You can
1. Replace `Stack` with Box and remove all props except `sx` (visual will remain unchanged), or
2. Change structure to
```
<Stack>
<Typography>Mysli na sebe i na mateřské dovolené.</Typography>
<Typography></Typography>
<Typography></Typography>
</Stack>
```